src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
 // write your code here
+const promise1 = new Promise((resolve, reject) => {
+  // додаю обробник кліку на елемент з класом лого
+  const logoElement = document.querySelector('.logo');
+
+  if (logoElement) {
+    logoElement.addEventListener('click', () => {
+      resolve('First promise was resolved');
+    });
+  } else {
+    reject(new Error('First promise not was resolved'));
+  }
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
```

Comment:
The error message here states 'First promise was rejected', which is misleading because this is actually the second promise. Consider changing the message to 'Second promise was rejected' for clarity.